PDKTSwipableCard/AppDelegate.swift

```diff
@@ -14,30 +14,30 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+	private func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
```

Comment:
Is there any particular reason to mark this function as `private`?

Reply:
I'm not quite sure 🤔. The auto-migrator marked it that way for some reason